src/models/People.js

```diff
@@ -103,13 +97,109 @@ static async displayPeopleProfessional() { // returns array of Timesheet Table
       defaultEntityTypeData = await db.executeStatement(defaultEntityTypeSQL, []);
       [{stringValue: defaultEntityType}] = defaultEntityTypeData.records[0];
  
-      return { session_token, user: {firstName, lastName, roles, capabilities, defaultEntityType, forcePasswordChange } };
-      
-    } catch (error) {
-      log(error.stack);
-      throw error; //throw error after logging so that the application handles the error
+      return { session_token, user: {firstName, lastName, groups, capabilities, defaultEntityType, forcePasswordChange} };
+  }
+
+  static async createUser({ firstName, lastName, loginEnabled, approved, email, password, forcePasswordChange }) {
+    const db = new DB();
+    const salt = await generateRandVals(5);
+
+    const sql = `INSERT INTO people (first_name, last_name, role, login_enabled, approved, email, encrypted_password, salt, created_at, force_password_change)
+                 VALUES (:first_name, :last_name, 'deprecated', :login_enabled, :approved, :email, SHA2(CONCAT(:password, :salt), 256), :salt, NOW(), :force_password_change)`;
+    const params = [
+       { name: 'first_name', value: { stringValue: firstName } },
+       { name: 'last_name', value: { stringValue: lastName } },
+       { name: 'login_enabled', value: { booleanValue: loginEnabled } },
+       { name: 'approved', value: { booleanValue: approved } },
+       { name: 'email', value: { stringValue: email } },
+       { name: 'password', value: { stringValue: password } },
+       { name: 'salt', value: { stringValue: salt } },
+       { name: 'force_password_change', value: { booleanValue: forcePasswordChange } }
+    ];
+
+    // Returns the id
+    const { generatedFields } = await db.exectuteStatement(sql, params);
```

Comment:
Please use the version with the correct speling. :)
```suggestion
    const { generatedFields } = await db.executeStatement(sql, params);
```

Reply:
Addressed: https://github.com/klaudsol/klaudsol-commons/pull/36/commits/9b00dd2514b3634ccfc617815bfd7b412ce86cbb